spring-data-sample/README.md

```diff
@@ -0,0 +1,262 @@
+Please see [this document](https://scalar-labs.github.io/scalardb-sql/spring-data-guide.html) for the details.
+
+### Schema
+
+[The schema](schema.json) is as follows:
+
+```json
+{
+  "sample.customers": {
+    "transaction": true,
+    "partition-key": [
+      "customer_id"
+    ],
+    "columns": {
+      "customer_id": "INT",
+      "name": "TEXT",
+      "credit_limit": "INT",
+      "credit_total": "INT"
+    }
+  },
+  "sample.orders": {
+    "transaction": true,
+    "partition-key": [
+      "customer_id"
+    ],
+    "clustering-key": [
+      "timestamp"
+    ],
+    "secondary-index": [
+      "order_id"
+    ],
+    "columns": {
+      "order_id": "TEXT",
+      "customer_id": "INT",
+      "timestamp": "BIGINT"
+    }
+  },
+  "sample.statements": {
+    "transaction": true,
+    "partition-key": [
+      "order_id"
+    ],
+    "clustering-key": [
+      "item_id"
+    ],
+    "columns": {
+      "order_id": "TEXT",
+      "item_id": "INT",
+      "count": "INT"
+    }
+  },
+  "sample.items": {
+    "transaction": true,
+    "partition-key": [
+      "item_id"
+    ],
+    "columns": {
+      "item_id": "INT",
+      "name": "TEXT",
+      "price": "INT"
+    }
+  }
+}
+```
+
+All the tables are created in the `sample` namespace.
+
+The `sample.customers` table manages customers' information.
+The `credit_limit` is the maximum amount of money a lender will allow each customer to spend using a credit card, and the `credit_total` is the amount of money that each customer has already spent by using the credit card.
+
+The `sample.orders` table manages orders' information, and the `sample.statements` table manages the statements' information of the orders. Finally, the `sample.items` table manages items' information to be ordered.
+
+
+The ER diagram for the schema is as follows:
+
+![ERD](images/ERD.png)
+
+### Transactions
+
+The following five transactions are implemented in this sample application:
+
+1. Getting customer information
+3. Placing an order. An order is paid by a credit card. It first checks if the amount of the money of the order exceeds the credit limit. If the check passes, it records order histories and updates the `credit_total`
+4. Getting order information by order ID
+5. Getting order information by customer ID
+6. Repayment. It reduces the amount of `credit_total`.
+
+### Configuration
+
+The configurations for the sample Spring Boot application is as follows:
+
+```application.properties
+spring.datasource.driver-class-name=com.scalar.db.sql.jdbc.SqlJdbcDriver
+spring.datasource.url=jdbc:scalardb:
+?scalar.db.storage=cassandra\
+&scalar.db.contact_points=localhost\
+&scalar.db.username=cassandra\
+&scalar.db.password=cassandra
+```
+
+Since you use Cassandra in this sample application as mentioned above, you need to configure Cassandra settings in the configuration.
```

Comment:
```suggestion
Since this sample application uses Cassandra, as shown above, you need to configure your settings for Cassandra in this configuration.
```